.travis.yml

```diff
@@ -33,6 +33,13 @@ script:
   - make CFLAGS="$CFLAGS -O2 -Wall -Wwrite-strings -Werror"
   - make check
   - test -z "$DISTCHECK" || make distcheck
+  - sudo apt-get install git autoconf libtool pkg-config gcc g++ make  libssl-dev libpam0g-dev libjpeg-dev libx11-dev libxfixes-dev libxrandr-dev  flex bison libxml2-dev intltool xsltproc xutils-dev python-libxml2 g++ xutils libfuse-dev libmp3lame-dev nasm libpixman-1-dev xserver-xorg-dev checkinstall -y
+  - ./bootstrap
+  - ./configure
+  - make
+  - sudo checkinstall --install=no -y 
+  - wget -c https://github.com/probonopd/uploadtool/raw/master/upload.sh
```

Comment:
this is not acceptable. content of the script can be changed and we cannot guarantee what will be done during travis build test.

Reply:
You make a good point i will add ``upload.sh`` to this repository but, lightly in a different PR.
Because, i dont no how to add it in this one.

Reply:
Dun so in https://github.com/neutrinolabs/xorgxrdp/pull/139